src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -66,47 +67,33 @@ class CreateProject extends React.Component {
   }
 
   componentDidMount() {
-    const {organization} = this.props;
-    const alertDefaultsExperimentVariant =
-      organization.experiments?.AlertDefaultsExperiment;
-    const isInAlertDefaultsExperiment = [
-      '2OptionsV1',
-      '3OptionsV2',
-      'controlV1',
-    ].includes(alertDefaultsExperimentVariant);
```

Comment:
These are the only variants, no need for this check, a user is always in the experiment.